modules/core/src/v2/coins/xtz.ts

```diff
@@ -112,17 +112,17 @@ export class Xtz extends BaseCoin {
   }
 
   /**
-   * Generate Tezos key pair
+   * Generate Tezos key pair - BitGo xpub format
    *
    * @param seed
-   * @returns {Object} object with generated pub, prv
+   * @returns {Object} object with generated xpub, xprv
    */
   generateKeyPair(seed?: Buffer): KeyPair {
     const keyPair = seed ? new bitgoAccountLib.Xtz.KeyPair({ seed }) : new bitgoAccountLib.Xtz.KeyPair();
-    const keys = keyPair.getKeys();
+    const keys = keyPair.getExtendedKeys();
     return {
-      pub: keys.pub,
-      prv: keys.prv!,
+      pub: keys.xpub,
+      prv: keys.xprv!,
```

Comment:
Why the `!`?

Reply:
yeah, I asked the same question.. this is more of an account-lib => consumers oddity. the prv is optional for this method but in typescript we want to say we always expect it 

https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci

Reply:
I'd rather throw in that case - the consumer of this method is expecting that the property is there.

Reply:
done